draft-ietf-scitt-architecture.md

```diff
@@ -748,12 +747,12 @@ The Transparency Service MUST also check the Signed Statement includes the requi
 The Transparency Service MAY validate the Signed Statement payload in order to enforce domain specific registration policies that apply to specific content types.
 1. **Apply Registration Policy:** The Transparency Service MUST check the attributes required by a Registration Policy are present in the protected headers.
   Custom Signed Statements are evaluated given the current Transparency Service state and the entire Envelope, and may use information contained in the attributes of named policies.
-1. **Register the Signed Statement** to the Append-only Log.
+1. **Register the Signed Statement** to the Verifiable Data Structure.
```

Comment:
```suggestion
1. **Register the Signed Statement** to the Append-only Log, and commit it to the Verifiable Data Structure.
```

Reply:
Really don't like this.
To register a signed statement is to obtain a receipt from a VDS.... some VDS are append only log.
SCITT can assert that only certain VDS are suitable for this use case.

Reply:
Please provide an alternative suggestion?

Reply:
Maybe just stop at "Register the Signed Statement"?

As Orie says above, the production of a receipt necessarily involves the use of one of the VDS options provided for in COSE Receipts, so removing words reduces redundancy, reduces opportunities for conflicts between references, and enables COSE Receipts to evolve usefully and independently.

Reply:
+1 Maybe just stop at "Register the Signed Statement"

Reply:
Amended the amendment accordingly.